Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -69,5 +74,25 @@ def commit(self):
             distances = inv(distances)
         self.send("Distances", distances)
 
+    def send_report(self):
+        items = OrderedDict()
+        norm, normopt = self.normalization_method, self.normalization_options
+        inv, invopt = self.inversion_method, self.inversion_options
+        if norm == 0:
```

Comment:
Ah, I finally see the structure now. You have:
```py
if norm:
    if inv:
        ...
    else:
        ...
else:
    if inv:
        ...
    else:
        ...
```
which is kind of like a tree of height 2 (3) that covers all four possiblities. So good so far.

But what do you think about linearizing that tree into just two consecutive checks: if the widget does inversion, report it; if it does normalization, report it. Could do this simply:
```py
parts = []
if inv:
    parts.append('inversion ({})'.format(invopt[inv][0])
if norm:
    parts.append('normalization ({})'.format(normopt[norm][0]))
self.report_items('Model parameters',
                  (('Transformation', ', '.join(parts).capitalize() or 'None'),))
```